lecture/SyntaxCheatSheet.js

```diff
@@ -1,21 +1,57 @@
 // Syntax Cheat sheet
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Please pull in my latest changes by 
`git fetch upstream master`
`git merge upstream/master`